### PR TITLE
router setup and redirecting is done

### PR DESCRIPTION
**Pull request by manusankar9**

### CODE REVIEW

**Comment by sivasankar9** [comment]
@manusankar9  Please fix build failure ASAP.

**Comment by manusankar9** [comment]
@sivasankar9 I've fixed build errors.

File: src/components/full-calender/components/shared/login/component/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            {(props.error || error) && (
+                <Alert severity='error' onClick={() => setError(null)}>
+                    {props.error || error}
+                </Alert>
+            )}
+
+        </form>
+    );
+};
+{/* <div> */ }
```
Comment (sivasankar9) [inline src/components/full-calender/components/shared/login/component/index.js]: Remove the commented code.

File: src/components/full-calender/components/shared/login/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Login from './component';
+import { connect } from "react-redux";
+import {login} from './actions'
+
+// const mapStateToProps = state =>  ({
```
Comment (sivasankar9) [inline src/components/full-calender/components/shared/login/index.js]: clean the commented code.

**Review by sivasankar9 — CHANGES_REQUESTED** [review]

**Comment by manusankar9** [comment]
@sivasankar9 I've fixed review comments

File: src/components/full-calender/components/shared/login/index.js (hunk 2 above)
Comment (manusankar9) [reply src/components/full-calender/components/shared/login/index.js]: removed the commented code

**Pull request merged**